Kiosk/Bid Fulfillment/Models/BidDetails.swift

```diff
@@ -39,7 +39,7 @@ import Moya
                 // Grab existing endpoint to piggy-back off of any existing configurations being used by the sharedprovider.
                 let endpoint = Networking.endpointsClosure()(target)
 
-                return endpoint.adding(newParameters: ["auction_pin": pin, "number": number, "sale_id": auctionID])
+                return endpoint.adding(newHTTPHeaderFields: ["auction_pin": pin, "number": number, "sale_id": auctionID])
```

Comment:
This changed from adding parameters to adding HTTP header fields, which our API doesn't handle in the same way. No biggie – looks like the `adding(newParameters:)` function was removed as part of Moya 9's migration to Tasks, so this was the only suggested function. I only point this out in case folks are wondering how to migrate the use of the old function; my attempt is here: https://github.com/artsy/eidolon/pull/678/files#diff-319cda9244d4aefd8caec7cc37a529c3R42 If anyone has a better way, let me know 👍